docs/content/concepts/partitions-schedules-sensors/sensors.mdx

```diff
@@ -159,6 +166,8 @@ def sensor_B():
 
 These sensor definitions are short, so they run in less than a second. Therefore, you can expect these sensors to run consistently around every 30 and 45 seconds, respectively.
 
+If a sensor evaluation function takes more than 60 seconds to return its results, the sensor evaluation will time out and the Dagster Daemon will move on to the next sensor without submitting any runs. This 60 second timeout only applies to the time it takes to run the sensor function, not to the execution time of the runs submitted by the sensor. To avoid timeouts, slower sensors can break up their work into chunks, using [cursors](/concepts/partitions-schedules-sensors/sensors#sensor-optimizations-using-cursors) to let subsequent sensor calls pick up where the previous call left off.
```

Comment:
i also snuck in a couple of answers to sensor questions that have been coming up frequently lately - we should also include the fact that they execute in the gRPC server /user code container

Reply:
This is great.  We may even want to put some sort of anchor tag here to link it?